### PR TITLE
Texturing: parameter renaming

### PR DESCRIPTION
**Pull request by natowi**
Texturing: parameter renaming: TextureSide -> TextureWidth

⚠️ changed parameter name

Connected to https://github.com/alicevision/meshroom/pull/903

### CODE REVIEW

**Comment by fabiencastan** [comment]
I agree that this name is not perfect, but `textureSize` is ambiguous as usually the size of an image is the number of pixels.
So we could maybe use `textureWidth`.

**Comment by natowi** [comment]
>  So we could maybe use textureWidth.

Sounds good. I will change it.

File: src/aliceVision/mesh/Texturing.cpp [hunk 1]
```diff
@@ -311,7 +311,7 @@ void Texturing::generateTexturesSubSet(const mvsUtils::MultiViewParams& mp,
     if(atlasIDs.size() > _atlases.size())
         throw std::runtime_error("Invalid atlas IDs ");
 
-    unsigned int textureSize = texParams.textureSide * texParams.textureSide;
+    unsigned int textureWidth = texParams.textureWidth * texParams.textureWidth;
```
Comment (fabiencastan) [inline src/aliceVision/mesh/Texturing.cpp:314]: `unsigned int textureSize = texParams.textureWidth * texParams.textureWidth;`

File: src/software/pipeline/main_texturing.cpp [hunk 2]
```diff
@@ -71,8 +71,8 @@ int aliceVision_main(int argc, char* argv[])
         ("imagesFolder", po::value<std::string>(&imagesFolder),
           "Use images from a specific folder instead of those specify in the SfMData file.\n"
           "Filename should be the image uid.")
-        ("textureSide", po::value<unsigned int>(&texParams.textureSide)->default_value(texParams.textureSide),
-            "Output texture size")
+        ("textureWidth", po::value<unsigned int>(&texParams.textureWidth)->default_value(texParams.textureSize),
```
Comment (fabiencastan) [inline src/software/pipeline/main_texturing.cpp:74]: Incorrect

**Pull request closed without merging**